state/lease/schema.go

```diff
@@ -172,13 +172,8 @@ func (doc clockDoc) skews(beginning, end time.Time) (map[string]Skew, error) {
 	// If it isn't, it could be ntp rolling the clock back slowly, so we add
```

Comment:
this comment and check are irrelevant now? we're using a monotonic clock, so by definition this cannot happen